tests/Doctrine/Tests/Common/Annotations/Fixtures/ClassWithPHPStanGenericsAnnotations.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Doctrine\Tests\Common\Annotations\Fixtures;
+
+/**
+ * @template T
+ */
+interface WithPHPStanExtendsAnnotationsInterface
+{
+
+}
+
+/**
+ * @template T
```

Comment:
`@template()` is still valid downstream, and changing rules around it is a major BC break at this point in time